internal/deviations/deviations.go

```diff
@@ -66,9 +66,32 @@ package deviations
 import (
 	"flag"
 
+	log "github.com/golang/glog"
+	"github.com/openconfig/featureprofiles/internal/metadata"
+	mpb "github.com/openconfig/featureprofiles/proto/metadata_go_proto"
 	"github.com/openconfig/ondatra"
 )
 
+// lookupDutDeviations returns the deviations for the specified dut. If no duts match `nil` is returned.
+func lookupDUTDeviations(dut *ondatra.DUTDevice) *mpb.Metadata_Deviations {
+	for _, platformExceptions := range metadata.Get().PlatformExceptions {
+		if platformExceptions.GetPlatform().Vendor.String() == dut.Device.Vendor().String() && platformExceptions.GetPlatform().HardwareModel == dut.Device.Model() {
+			return platformExceptions.GetDeviations()
+		}
+	}
+	log.Warningf("No platform exceptions for dut %v configured in platform exceptions metadata %v", dut, metadata.Get().PlatformExceptions)
+	return nil
+}
+
+// logErrorForFlag logs an error if specified flag is set.
+func logErrorForFlag(name string) {
```

Comment:
maybe a more descriptive name would be "logErrorIfFlagSet" and then I think you could delete the godoc comment, because the name would be self-explanatory.

Reply:
done